### PR TITLE
BG1 NPC Translation to Portuguese Brazil

### PR DESCRIPTION
**Pull request by Saurimxxm**
Hello, the translation of BG1NPC into Brazilian Portuguese or PtBr has already been completed and tested. I will make it available to put in the mod.

Merry Christmas and A Happy New Year 

[BG1NPC Portuges.zip](https://github.com/Gibberlings3/BG1NPC/files/13779735/BG1NPC.Portuges.zip)

### CODE REVIEW

File: bg1npc/tra/english/bg1npc_tmp.tra [hunk 1]
```diff
@@ -1,93 +1,93 @@
-@2    = ~[The Personal Diary of Sarevok]Diary
```
Comment (Roberciiik) [inline bg1npc/tra/english/bg1npc_tmp.tra:1]: It looks like you overwritten English translation.

**Comment by Gitjas** [comment]
Thank! I don't know how, but your commits ended up creating 140 Email notifications. Maybe next time finish your changes in your repo and then make one pull request for all commits. Cheers!

**Comment by Gitjas** [comment]
Wait - did this replace existing files like in EbdlessBG1?

**Comment by Saurimxxm** [comment]
> Wait - did this replace existing files like in EndlessBG1?

I only realized this small error later and I even tried to undo what I did but I couldn't.
I don't know how to use GitHub, it's complex and unintuitive (at least for me) that's why I made the translated file available

**Comment by Gitjas** [comment]
Saurimxxm, I reverted your changes. 
- The tra files need to be in ANSI format; the English files you restored didn't seem to be.
- To add the new translation, create a folder in bg1npc/tra for the Brazilian Portuguese language where you put in all the tra files. I'm not sure what you added, but it was not a folder. Do not work on the web interface of GitHub directly as it scrambles the file format. Best use a tool like GitHub Desktop which allows you to work on your computer.
- Please, make one commit for the translation, not commits for every file each. It clutters the history otherwise.

**Comment by Gitjas** [comment]
Yes, GitHub is not very user-friendly. Thanks for trying! Fortunately, it can all be reversed, that's one of the advantages of usin GitHub. Don't give up! :-) (I managed to understand it, sort of, so it's not impossible. ;-) )

**Pull request merged**